nifi-nar-bundles/nifi-graph-bundle/nifi-graph-processors/src/test/java/org/apache/nifi/processors/graph/util/SimpleEntry.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.nifi.processors.graph.util;
+
+import java.util.Map;
+
+final class SimpleEntry<K, V> implements Map.Entry<K, V> {
```

Comment:
See if you can use [this](https://docs.oracle.com/javase/8/docs/api/java/util/AbstractMap.SimpleEntry.html) 

Syntax to import it is `import static java.util.AbstractMap.SimpleEntry`

Reply:
Yup, was able to use that! Thanks for that suggestion.